app/src/main/java/ch/epfl/qedit/backend/auth/AuthenticationService.java

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.qedit.backend.auth;
+
+import ch.epfl.qedit.model.User;
+import ch.epfl.qedit.util.Callback;
+import ch.epfl.qedit.util.Response;
+
+/**
+ * Represents an authentication service, that is a service to which login requests can be sent. It
+ * handles asynchronous responses by using a callback.
+ */
+public interface AuthenticationService {
+    int CONNECTION_ERROR = 1;
```

Comment:
Maybe we could use an enum for those ERRORs. I have seen as well that you use them in other classes. Same things for the WRONG_STUFFs

Reply:
Yes, initially that's what I did. The problem is that different services can have different errors, so making an enum would mean that all those errors would be grouped in a single place, which I think doesn't make sense.

Reply:
Yes i first thinked like you Cosme and then i understood why antoine did it that way.

Reply:
I agree different services can have different errors but to me it seems that AuthentificationService and Response do have the same kinds. 

By the same way using int doesn't make sense to me.

Reply:
Responce is generic and is used by AuthService and DatabaseService, but we could use it in different context (other than backend related stuff) i think

Reply:
Do you receive this comments?